app/pages/index.js

```diff
@@ -9,6 +9,8 @@ import Growthcounters from '../components/growthcounters';
 import { Container, Col } from 'react-bootstrap';
 import { fetchAPI } from '../lib/api';
 import { withFirebaseAuthUser } from '../components/auth/firebase';
+import GithubIssuesList from '../components/githubissueslist';
+import { octokitClient } from '../lib/octokit'
```

Comment:
```suggestion
import { getIssues } from '../lib/github'
```

Consider these just suggestions, do it your own way, no need to commit these exact ones.